pkg/server/helpers.go

```diff
@@ -320,9 +319,9 @@ func (c *criContainerdService) localResolve(ctx context.Context, ref string) (*m
 		}
 		image, err := c.imageStoreService.Get(ctx, normalized.String())
 		if err != nil {
-			if images.IsNotFound(err) {
-				return nil, nil
-			}
+			//if images.IsNotFound(err) { // TODO (mikebrow): IsNotFound removed
```

Comment:
Seems moved to https://github.com/containerd/containerd/blob/bdf9f5f7388e8203e63a74b89800f7f3dd4a7743/metadata/errors.go.